src/components/ButtonBase/ButtonBase.style.ts

```diff
@@ -1,51 +1,51 @@
 import { css, SerializedStyles } from '@emotion/react';
-import { rem } from 'theme/utils';
 
-import { Theme } from '../../theme';
-import { getDisabled, getFocus, getHover, getPressed } from '../../theme/states';
+import stateTokens from '../../theme/states/states.tokens';
+import buttonTokens from '../Button/Button.tokens';
 import { ButtonBaseProps } from './ButtonBase';
 
-export const buttonBaseStyle =
-  ({ isBlock, sx }: Omit<ButtonBaseProps, 'buttonType' | 'ref'>) =>
-  (theme: Theme): SerializedStyles => {
-    /** @TODO replace all temporary css values with the new ones */
-
-    const backGroundColor = '#123456';
-    const borderWidth = 0;
+export const buttonWrapperStyle = ({
+  isBlock,
+}: Pick<ButtonBaseProps, 'isBlock'>): SerializedStyles =>
+  css({ position: 'relative', width: isBlock ? '100%' : 'fit-content' });
 
+export const buttonBaseStyle =
+  ({
```

Comment:
i expect that we miss the `circular` functionality here that the icon button needs

Reply:
this will be added in a different, dedicated for the IconButton PR